src/modules/rawdata/services/interestPoint/ListInterestPointService.ts

```diff
@@ -0,0 +1,23 @@
+import { injectable, inject } from 'tsyringe';
```

Comment:
change the file name to IndexInterestPointService, to maintain verbal standards

Reply:
I changed the name both here and in the "ListInterestPointTypeService", thanks